man/dracut.8.adoc

```diff
@@ -96,6 +96,10 @@ example:
 **-o, --omit** _<list of dracut modules>_::
     Omit a space-separated list of dracut modules. This parameter can be
     specified multiple times.
+    Best to avoid omitting dracut modules as a user as you might be
```

Comment:
The phrase "Best to avoid omitting" made me stumble reading it - multiple times (due to the double negation). Could that maybe rephrased?